<ui version="4.0" >
 <author>rncbc aka Rui Nuno Capela</author>
 <comment>qtractor - An Audio/MIDI multi-track sequencer.

   Copyright (C) 2005-2007, rncbc aka Rui Nuno Capela. All rights reserved.

   This program is free software; you can redistribute it and/or
   modify it under the terms of the GNU General Public License
   as published by the Free Software Foundation; either version 2
   of the License, or (at your option) any later version.

   This program is distributed in the hope that it will be useful,
   but WITHOUT ANY WARRANTY; without even the implied warranty of
   MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
   GNU General Public License for more details.

   You should have received a copy of the GNU General Public License along
   with this program; if not, write to the Free Software Foundation, Inc.,
   51 Franklin Street, Fifth Floor, Boston, MA 02110-1301 USA.

  </comment>
 <exportmacro></exportmacro>
 <class>qtractorSessionForm</class>
 <widget class="QDialog" name="qtractorSessionForm" >
  <property name="geometry" >
   <rect>
    <x>0</x>
    <y>0</y>
    <width>400</width>
    <height>458</height>
   </rect>
  </property>
  <property name="focusPolicy" >
   <enum>Qt::StrongFocus</enum>
  </property>
  <property name="windowTitle" >
   <string>Session - Qtractor</string>
  </property>
  <property name="windowIcon" >
   <iconset resource="../icons/qtractor.qrc" >:/icons/qtractor.png</iconset>
  </property>
  <layout class="QVBoxLayout" >
   <property name="margin" >
    <number>9</number>
   </property>
   <property name="spacing" >
    <number>6</number>
   </property>
   <item>
    <widget class="QLabel" name="SessionNameTextLabel" >
     <property name="text" >
      <string>&amp;Name:</string>
     </property>
     <property name="buddy" >
      <cstring>SessionNameLineEdit</cstring>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QLineEdit" name="SessionNameLineEdit" >
     <property name="minimumSize" >
      <size>
       <width>320</width>
       <height>0</height>
      </size>
     </property>
     <property name="toolTip" >
      <string>Session name</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QLabel" name="SessionDirTextLabel" >
     <property name="text" >
      <string>&amp;Directory:</string>
     </property>
     <property name="buddy" >
      <cstring>SessionDirComboBox</cstring>
     </property>
    </widget>
   </item>
   <item>
    <layout class="QHBoxLayout" >
     <property name="margin" >
      <number>0</number>
     </property>
     <property name="spacing" >
      <number>4</number>
     </property>
     <item>
      <widget class="QComboBox" name="SessionDirComboBox" >
       <property name="minimumSize" >
        <size>
         <width>320</width>
         <height>0</height>
        </size>
       </property>
       <property name="toolTip" >
        <string>Session directory</string>
       </property>
       <property name="editable" >
        <bool>true</bool>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QToolButton" name="SessionDirToolButton" >
       <property name="minimumSize" >
        <size>
         <width>22</width>
         <height>22</height>
        </size>
       </property>
       <property name="maximumSize" >
        <size>
         <width>24</width>
         <height>24</height>
        </size>
       </property>
       <property name="focusPolicy" >
        <enum>Qt::TabFocus</enum>
       </property>
       <property name="toolTip" >
        <string>Browse for session directory</string>
       </property>
       <property name="text" >
        <string>...</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <widget class="QLabel" name="DescriptionTextLabel" >
     <property name="text" >
      <string>&amp;Description:</string>
     </property>
     <property name="buddy" >
      <cstring>DescriptionTextEdit</cstring>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QTextEdit" name="DescriptionTextEdit" >
     <property name="toolTip" >
      <string>Session description</string>
     </property>
    </widget>
   </item>
   <item>
    <layout class="QHBoxLayout" >
     <property name="margin" >
      <number>0</number>
     </property>
     <property name="spacing" >
      <number>6</number>
     </property>
     <item>
      <widget class="QGroupBox" name="TimeGroupBox" >
       <property name="sizePolicy" >
        <sizepolicy>
         <hsizetype>5</hsizetype>
         <vsizetype>1</vsizetype>
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="title" >
        <string>Time</string>
       </property>
       <layout class="QGridLayout" >
        <property name="margin" >
         <number>9</number>
        </property>
        <property name="spacing" >
         <number>6</number>
        </property>
        <item row="3" column="0" >
         <widget class="QLabel" name="TicksPerBeatTextLabel" >
          <property name="text" >
           <string>T&amp;icks/Beat:</string>
          </property>
          <property name="alignment" >
           <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
          </property>
          <property name="buddy" >
           <cstring>TicksPerBeatSpinBox</cstring>
          </property>
         </widget>
        </item>
        <item row="2" column="0" >
         <widget class="QLabel" name="BeatsPerBarTextLabel" >
          <property name="text" >
           <string>&amp;Beats/Bar:</string>
          </property>
          <property name="alignment" >
           <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
          </property>
          <property name="buddy" >
           <cstring>BeatsPerBarSpinBox</cstring>
          </property>
         </widget>
        </item>
        <item row="1" column="0" >
         <widget class="QLabel" name="TempoTextLabel" >
          <property name="text" >
           <string>&amp;Tempo:</string>
          </property>
          <property name="alignment" >
           <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
          </property>
          <property name="buddy" >
           <cstring>TempoSpinBox</cstring>
          </property>
         </widget>
        </item>
        <item row="0" column="0" >
         <widget class="QLabel" name="SampleRateTextLabel" >
          <property name="text" >
           <string>Sample &amp;Rate:</string>
          </property>
          <property name="alignment" >
           <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
          </property>
          <property name="buddy" >
           <cstring>SampleRateComboBox</cstring>
          </property>
         </widget>
        </item>
        <item row="2" column="1" >
         <widget class="QSpinBox" name="BeatsPerBarSpinBox" >
          <property name="toolTip" >
           <string>Time signature (beats/bar)</string>
          </property>
          <property name="suffix" >
           <string> / 4</string>
          </property>
          <property name="maximum" >
           <number>32</number>
          </property>
          <property name="minimum" >
           <number>2</number>
          </property>
          <property name="value" >
           <number>4</number>
          </property>
         </widget>
        </item>
        <item row="1" column="1" >
         <widget class="QDoubleSpinBox" name="TempoSpinBox" >
          <property name="toolTip" >
           <string>Tempo (BPM)</string>
          </property>
          <property name="decimals" >
           <number>1</number>
          </property>
          <property name="maximum" >
           <double>1000</double>
          </property>
          <property name="minimum" >
           <double>1</double>
          </property>
          <property name="singleStep" >
           <double>0.1</double>
          </property>
          <property name="value" >
           <double>120</double>
          </property>
         </widget>
        </item>
        <item row="3" column="1" >
         <widget class="QSpinBox" name="TicksPerBeatSpinBox" >
          <property name="toolTip" >
           <string>Resolution (ticks/beat; TPQN)</string>
          </property>
          <property name="maximum" >
           <number>600</number>
          </property>
          <property name="minimum" >
           <number>12</number>
          </property>
          <property name="singleStep" >
           <number>12</number>
          </property>
          <property name="value" >
           <number>96</number>
          </property>
         </widget>
        </item>
        <item row="0" column="1" >
         <widget class="QComboBox" name="SampleRateComboBox" >
          <property name="toolTip" >
           <string>Sample rate (Hz)</string>
          </property>
          <property name="editable" >
           <bool>true</bool>
          </property>
          <item>
           <property name="text" >
            <string>44100</string>
           </property>
          </item>
          <item>
           <property name="text" >
            <string>48000</string>
           </property>
          </item>
          <item>
           <property name="text" >
            <string>96000</string>
           </property>
          </item>
          <item>
           <property name="text" >
            <string>192000</string>
           </property>
          </item>
         </widget>
        </item>
       </layout>
      </widget>
     </item>
     <item>
      <widget class="QGroupBox" name="ViewGroupBox" >
       <property name="sizePolicy" >
        <sizepolicy>
         <hsizetype>5</hsizetype>
         <vsizetype>1</vsizetype>
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="title" >
        <string>View</string>
       </property>
       <layout class="QGridLayout" >
        <property name="margin" >
         <number>9</number>
        </property>
        <property name="spacing" >
         <number>6</number>
        </property>
        <item row="0" column="1" >
         <widget class="QComboBox" name="SnapPerBeatComboBox" >
          <property name="toolTip" >
           <string>Snap/beat</string>
          </property>
          <property name="editable" >
           <bool>false</bool>
          </property>
          <item>
           <property name="text" >
            <string>None</string>
           </property>
          </item>
          <item>
           <property name="text" >
            <string>Beat</string>
           </property>
          </item>
          <item>
           <property name="text" >
            <string>Beat/2</string>
           </property>
          </item>
          <item>
           <property name="text" >
            <string>Beat/4</string>
           </property>
          </item>
          <item>
           <property name="text" >
            <string>Beat/8</string>
           </property>
          </item>
          <item>
           <property name="text" >
            <string>Beat/16</string>
           </property>
          </item>
          <item>
           <property name="text" >
            <string>Beat/32</string>
           </property>
          </item>
         </widget>
        </item>
        <item row="1" column="1" >
         <widget class="QSpinBox" name="PixelsPerBeatSpinBox" >
          <property name="toolTip" >
           <string>Pixels/beat</string>
          </property>
          <property name="maximum" >
           <number>240</number>
          </property>
          <property name="minimum" >
           <number>4</number>
          </property>
          <property name="singleStep" >
           <number>4</number>
          </property>
          <property name="value" >
           <number>32</number>
          </property>
         </widget>
        </item>
        <item rowspan="2" row="3" column="1" >
         <widget class="QSpinBox" name="VerticalZoomSpinBox" >
          <property name="toolTip" >
           <string>Vertical Zoom (%)</string>
          </property>
          <property name="suffix" >
           <string> %</string>
          </property>
          <property name="maximum" >
           <number>1000</number>
          </property>
          <property name="minimum" >
           <number>10</number>
          </property>
          <property name="singleStep" >
           <number>10</number>
          </property>
          <property name="value" >
           <number>100</number>
          </property>
         </widget>
        </item>
        <item row="4" column="0" >
         <widget class="QLabel" name="VerticalZoomTextLabel" >
          <property name="text" >
           <string>&amp;Vertical Zoom:</string>
          </property>
          <property name="alignment" >
           <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
          </property>
          <property name="buddy" >
           <cstring>VerticalZoomSpinBox</cstring>
          </property>
         </widget>
        </item>
        <item row="2" column="1" >
         <widget class="QSpinBox" name="HorizontalZoomSpinBox" >
          <property name="toolTip" >
           <string>Horizontal Zoom (%)</string>
          </property>
          <property name="suffix" >
           <string> %</string>
          </property>
          <property name="maximum" >
           <number>1000</number>
          </property>
          <property name="minimum" >
           <number>10</number>
          </property>
          <property name="singleStep" >
           <number>10</number>
          </property>
          <property name="value" >
           <number>100</number>
          </property>
         </widget>
        </item>
        <item rowspan="2" row="2" column="0" >
         <widget class="QLabel" name="HorizontalZoomTextLabel" >
          <property name="text" >
           <string>&amp;Horizontal Zoom:</string>
          </property>
          <property name="alignment" >
           <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
          </property>
          <property name="buddy" >
           <cstring>HorizontalZoomSpinBox</cstring>
          </property>
         </widget>
        </item>
        <item row="1" column="0" >
         <widget class="QLabel" name="PixelsPerBeatTextLabel" >
          <property name="text" >
           <string>&amp;Pixels/Beat:</string>
          </property>
          <property name="alignment" >
           <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
          </property>
          <property name="buddy" >
           <cstring>PixelsPerBeatSpinBox</cstring>
          </property>
         </widget>
        </item>
        <item row="0" column="0" >
         <widget class="QLabel" name="SnapPerBeatTextLabel" >
          <property name="text" >
           <string>&amp;Snap/Beat:</string>
          </property>
          <property name="alignment" >
           <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
          </property>
          <property name="buddy" >
           <cstring>SnapPerBeatComboBox</cstring>
          </property>
         </widget>
        </item>
       </layout>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <layout class="QHBoxLayout" >
     <property name="margin" >
      <number>0</number>
     </property>
     <property name="spacing" >
      <number>4</number>
     </property>
     <item>
      <spacer>
       <property name="orientation" >
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeType" >
        <enum>QSizePolicy::Expanding</enum>
       </property>
       <property name="sizeHint" >
        <size>
         <width>20</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <widget class="QPushButton" name="OkPushButton" >
       <property name="text" >
        <string>OK</string>
       </property>
       <property name="icon" >
        <iconset resource="../icons/qtractor.qrc" >:/icons/formAccept.png</iconset>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="CancelPushButton" >
       <property name="text" >
        <string>Cancel</string>
       </property>
       <property name="icon" >
        <iconset resource="../icons/qtractor.qrc" >:/icons/formReject.png</iconset>
       </property>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <layoutdefault spacing="4" margin="8" />
 <pixmapfunction></pixmapfunction>
 <tabstops>
  <tabstop>SessionNameLineEdit</tabstop>
  <tabstop>SessionDirComboBox</tabstop>
  <tabstop>SessionDirToolButton</tabstop>
  <tabstop>DescriptionTextEdit</tabstop>
  <tabstop>SampleRateComboBox</tabstop>
  <tabstop>TempoSpinBox</tabstop>
  <tabstop>BeatsPerBarSpinBox</tabstop>
  <tabstop>TicksPerBeatSpinBox</tabstop>
  <tabstop>SnapPerBeatComboBox</tabstop>
  <tabstop>PixelsPerBeatSpinBox</tabstop>
  <tabstop>HorizontalZoomSpinBox</tabstop>
  <tabstop>VerticalZoomSpinBox</tabstop>
  <tabstop>OkPushButton</tabstop>
  <tabstop>CancelPushButton</tabstop>
 </tabstops>
 <resources>
  <include location="../icons/qtractor.qrc" />
 </resources>
 <connections/>
</ui>
